junit-jupiter-api/src/main/java/org/junit/jupiter/api/io/TempDir.java

```diff
@@ -73,4 +73,11 @@
 @Documented
 @API(status = EXPERIMENTAL, since = "5.4")
 public @interface TempDir {
+	/**
+	 * Returns an identifier for temp directory.
+	 *
+	 * @return an identifier for the underlying temp directory, where an empty String represents the old identifier
```

Comment:
Could you please more in depth describe what the identifier is for?
Additionally it is most likely not clear to the users what "the old identifier" means.

Reply:
@Marcono1234 Thanks for the review! I've improved the Javadoc in one of the latest commits. PTAL!